src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/stønadsvilkår/regler/evalutation/OppdaterVilkår.kt

```diff
@@ -186,9 +186,13 @@ object OppdaterVilkår {
             }
         }
 
-    private fun utledStatus(eksisterendeVilkår: Vilkår): VilkårStatus? =
-        when (eksisterendeVilkår.status) {
-            VilkårStatus.UENDRET -> VilkårStatus.ENDRET
+    private fun utledStatus(
+        eksisterendeVilkår: Vilkår,
+        lagreVilkårDto: LagreVilkårDto,
+    ): VilkårStatus? =
+        when {
+            eksisterendeVilkår.erFremtidigUtgift && lagreVilkårDto.erFremtidigUtgift != true -> VilkårStatus.NY
+            eksisterendeVilkår.status == VilkårStatus.UENDRET -> VilkårStatus.ENDRET
```

Comment:
```suggestion
    private fun utledStatus(
        eksisterendeVilkår: Vilkår,
        oppdatering: LagreVilkårDto,
    ): VilkårStatus? =
        when {
            eksisterendeVilkår.erFremtidigUtgift && oppdatering.erFremtidigUtgift != true -> VilkårStatus.NY
            eksisterendeVilkår.status == VilkårStatus.UENDRET -> VilkårStatus.ENDRET
```

Syntes bare det går 3 ms raskere å lese koden sånn 🦾